3기/HyeonkyeongSeo/3주차/index.html

```diff
@@ -0,0 +1,230 @@
+                                            </li>
+                                        </ul>
+                                    </div>
+                                    <div class="portfolio_detail_bottom">
+                                        <p></p>
+                                    </div>
+                                </div>
+                            </div>
+-->
+                        </ol>
+                    </div>
+                </div>
+                
+                <div class="contact">
+                    <div class="wrap">
+                        <div class="contact_headtext headtext">
+                            <h2>GET IN TOUCH</h2>
+                            <p class="contact_head headp">연락하기</p>
+                        </div>
+                        <div id="contact_area" class="contents_area">
+                            <ul class="contact_info">
+                                <li><img src="images/contact_mail.png" alt="이메일아이콘"><a href="mailto:komaof1004@naver.com" class="underline">komaof1004@naver.com</a></li>
+                                <li><img src="images/contact_phone_enabled.png" alt="전화아이콘"><a href="tel:010-2278-0913" class="underline">010-2278-0913</a></li>
+                                <li><img src="images/contact_weblink.png" alt="링크아이콘"><a href="https://neomi-design-lab.tistory.com/" class="underline">https://neomi-design-lab.tistory.com/</a></li>
+                                <li><img src="images/contact_location_on.png" alt="장소아이콘"><a href="">서울시 중랑구</a></li>
+                            </ul>
+                            <div class="contact_form">
+                               <div class="form_area">
+                                   <ul class="form_list">
+                                        <li><input type="text" placeholder="NAME" required></li>
+                                        <li><input type="text" placeholder="EMAIL" required></li>
+                                        <li><textarea name="message" id="contact_msg" placeholder="MASSAGE" required></textarea></li>
+                                    </ul>
+                                    <button type="button" class="send_btn">SUBMIT →</button>
+                               </div>
+                                <div class="formimg_box">
+                                    <img src="images/contact_local_post_office.png" alt="우체통아이콘" class="formimg">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+        </main>
+
+        <footer id="footer">
+            <div class="wrap">
+                <p class="footer_pctab">Copyright 2023 <strong>Neomi.seo</strong> Allrights Reserved.</p>
+                <p class="footer_mobile">ⓒ Neomi.seo</p>
+            </div>
+        </footer>
+    </div>
+    
+    <script>
+        
+        fetch('../json/timeline.json')
```

Comment:
index.html 파일과 json 폴더는 동일 선상의 경로에 있기 때문에, `../` 이렇게 상위 폴더로 나가서 찾는게 아닌 `./`현재 위치 기준으로 찾아가주셔야해요!

```suggestion
        fetch('./json/timeline.json')
```

Reply:
아래 portfolio.json도 동일한 피드백 드려요 😊